### PR TITLE
Updated CDN template and readme

### PR DESCRIPTION
**Pull request by asudbring**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by asudbring** [comment]
Not sure why the Govt one failed, might need to be rerun on the agent side as it has some weird errors.
#sign-off

**Comment by asudbring** [comment]
@bmoore-msft any chance to get this merged today?  

Thanks!
allen

File: 101-cdn-with-custom-origin/README.md [hunk 1]
```diff
@@ -9,8 +9,29 @@
...
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cdn-with-custom-origin%2Fazuredeploy.json)
+  
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cdn-with-custom-origin%2Fazuredeploy.json)
```
Comment (bmoore-msft) [inline 101-cdn-with-custom-origin/README.md:14]: remove this if it's not supported in AzureGov

File: 101-cdn-with-custom-origin/azuredeploy.parameters.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
             "value": "GEN-UNIQUE"
         },
         "originUrl": {
-            "value": "www.microsoft.com"
+            "value": "cdndemo.azurewebsites.net"
```
Comment (bmoore-msft) [inline 101-cdn-with-custom-origin/azuredeploy.parameters.json:12]: revert this change (or use a web site that can't be removed)

File: 101-cdn-with-custom-origin/README.md (hunk 1 above)
Comment (asudbring) [reply 101-cdn-with-custom-origin/README.md:14]: It is supported on Azure GOV, it's just a CDN profile.  Should I still remove it?

File: 101-cdn-with-custom-origin/azuredeploy.parameters.json (hunk 2 above)
Comment (asudbring) [reply 101-cdn-with-custom-origin/azuredeploy.parameters.json:12]: This was created specifically for CDN for demos and is being used by our current QS.  I went ahead and changed it back.

File: 101-cdn-with-custom-origin/README.md (hunk 1 above)
Comment (asudbring) [reply 101-cdn-with-custom-origin/README.md]: Was able to fix errors in readme and it builds clean in both environments.

**Comment by asudbring** [comment]
@bmoore-msft fixed errors and it builds clean in both environments.

please merge.

Thanks!
allen

**Comment by asudbring** [comment]
@bmoore-msft anyway I can get this merged?  We are working on some deadlines for ARM templates for articles.  

Thanks!
allen

**Comment by mgblythe** [comment]
@bmoore-msft - checking if there's anything blocking or whether you can merge this one, thanks!
